xmind/tests/test_markerref_markerid.py

```diff
@@ -0,0 +1,131 @@
+from xmind.core.markerref import MarkerId
+import logging
+from . import base
+
+
+class MarkerrefTest(base.Base):
+    """Markerref test"""
+    LOGGER = logging.getLogger('MarkerefTest')
+
+    def test_init_set_name(self, name='test-name'):
```

Comment:
What is the reason to have test_ function with parameter? I would declare either just variable inside the method or even would go without any variable for such simple test.
Like 
```
def test_bla_bla_bla(self): 
el = MarkerId('some')
self.assertEqual(el.name, 'some')```